src/router/bundle/bundleHandler.test.ts

```diff
@@ -54,6 +54,9 @@ const genericResource: GenericResource = {
     typeHistory: stubs.history,
     typeSearch: stubs.search,
 };
+
+const dummyServerUrl: string = 'https://dummy-server-url';

```

Comment:
How are these broken?

Reply:
These are broken before this code change. It's fixed in this PR but not relevant to the change here.

Reply:
talked offline this is a feature branch